### DISCUSSION TITLE
More languages

### DISCUSSION THREAD

**Discussion by 3378644629** [community/community#34336] [Other Feature Feedback, Questions, & Ideas]
Can I have a simplified Chinese translation? I can't understand

**Comment by 3378644629** [comment]
（I'm not good at English）

**Comment by fadyat** [comment]
Hey! 😃 
You can download the extension for your browser, that translates selected text
For example: google translate extension for Google Chrome

**Comment by devhns** [comment]
Hi! :) If you wanna quick translate for any language to your main language, you can use just right click and select Translate to Chinese, this is simply in Google Chrome. If it's not work, you can download some extentions on Chrome Store ([examples here](https://chrome.google.com/webstore/search/translator?hl=zh)) to quickly translate your texts.
